pom.xml

```diff
@@ -72,49 +72,50 @@
         <scala.binary.version>2.12</scala.binary.version>
 
         <avro.version>1.11.3</avro.version>
-        <guava.version>32.1.2-jre</guava.version>
+        <guava.version>32.1.3-jre</guava.version>
 
         <!-- versions from geomesa -->
         <gt.version>28.2</gt.version>
         <accumulo.version>2.0.1</accumulo.version>
         <thrift.version>0.12.0</thrift.version>
-        <zookeeper.version>3.5.10</zookeeper.version>
```

Comment:
was there any 3.5.x version that fixed the bug? i think there are some incompatibilities between these 2 versions...

Reply:
Unfortunately no, this was the earliest version that had the fix.